.github/workflows/publish_image.yml

```diff
@@ -0,0 +1,34 @@
+name: Publish Docker image
+
+on:
+  push:
+    tags:
+      - '*'
+
+defaults:
+  run:
+    shell: bash
+
+jobs:
+  push_to_registry:
+    name: Push Docker image to OSC Container Registry
+    runs-on: ubuntu-latest
+    steps:
+      - name: checkout
+        uses: actions/checkout@v2
+      - name: setup the environment
+        run: |
+          TAG=${GITHUB_REF#refs/*/}
+          REPO="docker-registry.osc.edu/gateways"
+          echo "LATEST_IMG=$REPO/phylogatr:latest" >> $GITHUB_ENV
```

Comment:
I have a vague recollection that this is rejected by Github actions due to security.  This is how you save values:

```
echo ::set-output name=latest::$LATEST_IMG
echo ::set-output name=version::$VERSIONED_IMG
```

Then you reference like this:

```
${{ steps.setup.outputs.latest }}
${{ steps.setup.outputs.version }}
```

The step doing setup would need this:

```
id: setup
```

I think it's a lot simpler to just setup output for tag version and reference that and do like 

```
docker push docker-registry.osc.edu/gateways/phylogatr:${{ steps.setup.outputs.version }}
docker push docker-registry.osc.edu/gateways/phylogatr:latest
```